Week2/homework/joins.js

```diff
@@ -0,0 +1,28 @@
+var mysql = require('mysql');
+var connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'hyfuser',
+    password: 'hyfpassword',
+    database: 'quarantine'
+});
+
+connection.connect();
+connection.query(`SELECT Employee.full_name AS Employee, 
+manager.full_name AS manager 
+FROM Employee
+LEFT JOIN Employee AS manager 
+ON Employee.manager_id = manager.employee_no`, function (error, results, fields) {
+    if (error) throw error;
+    console.log('Join created');
+});
+connection.query(`SELECT Employee.full_name AS Employee, 
+Department.title 
+FROM Employee
+LEFT JOIN Department 
```

Comment:
Since you're placing `Employee` on the right, and `Department` on the left (`FROM Employee JOIN Department`), I think you need a `RIGHT JOIN` in order to also include departments without any employees?